src/components/cart.lineitem.less

```diff
@@ -89,81 +99,58 @@
     padding-left: 30px;
   }
 
-  .cart-lineitem-thumbnail-col img {
-    max-width: 60px;
-    background-color: #eee;
-    padding: 2px;
-    border-radius: 2px;
-    border: 1px solid #dddddd;
-    background: #ffffff;
-    /* Old browsers */
-
-    background: -moz-linear-gradient(-45deg, #ffffff 0%, #ffffff 100%);
-    /* FF3.6+ */
-
-    background: -webkit-gradient(linear, left top, right bottom, color-stop(0%, #ffffff), color-stop(100%, #ffffff));
-    /* Chrome,Safari4+ */
-
-    background: -webkit-linear-gradient(-45deg, #ffffff 0%, #ffffff 100%);
-    /* Chrome10+,Safari5.1+ */
-
-    background: -o-linear-gradient(-45deg, #ffffff 0%, #ffffff 100%);
-    /* Opera 11.10+ */
-
-    background: -ms-linear-gradient(-45deg, #ffffff 0%, #ffffff 100%);
-    /* IE10+ */
-
-    background: linear-gradient(135deg, #ffffff 0%, #ffffff 100%);
-    /* W3C */
-
-    -webkit-box-shadow: inset 0px 0px 6px 2px rgba(255, 255, 255, 0.5);
-    box-shadow: inset 0px 0px 6px 2px rgba(255, 255, 255, 0.5);
-    -webkit-box-shadow: 0px 2px 0px 0px rgba(0, 0, 0, 0.2);
-    box-shadow: 0px 2px 0px 0px rgba(0, 0, 0, 0.2);
-    box-shadow: none;
```

Comment:
`box-shadow: none` is the default and it overrides previous lines anyway, so this whole piece can be removed.